docs/source/guide/sdk.md

```diff
@@ -0,0 +1,175 @@
+```python
+project.create_prediction(task_ids[1], result={"x": 10, "y": 20, "width": 30, "height": 40, "label": ["Dog"]}, score=0.9)
+```
+
+For another example, see the [Jupyter notebook example of importing pre-annotated data](https://github.com/heartexlabs/label-studio-sdk/blob/master/examples/Import%20preannotations.ipynb).
+
+### Import pre-annotated tasks into Label Studio
+
+You can also import predictions together with tasks as pre-annotated tasks. The SDK offers several ways that you can import pre-annotations into Label Studio.
+
+One way is to import tasks in a simple JSON format, where one key in the JSON identifies the data object being labeled, and the other is the key containing the prediction. 
+
+In this example, import predictions for an image classification task:
+```python
+project.import_tasks(
+    [{'image': f'https://data.heartex.net/open-images/train_0/mini/0045dd96bf73936c.jpg', 'pet': 'Dog'},
+    {'image': f'https://data.heartex.net/open-images/train_0/mini/0083d02f6ad18b38.jpg', 'pet': 'Cat'}],
+    preannotated_from_fields=['pet']
+)
+```
+The image is specified in the `image` key using a public URL, and the prediction is referenced in an arbitrary `pet` key, which is then specified in the `preannotated_from_fields()` method.  
+
+For more examples, see the [Jupyter notebook example of importing pre-annotated data](https://github.com/heartexlabs/label-studio-sdk/blob/master/examples/Import%20preannotations.ipynb).
+
+## Prepare and manage data with filters
+
+You can also use the SDK to control how tasks appear in the data manager to annotators or reviewers. You can create custom filters and ordering for the tasks based on parameters that you specify with the SDK. This lets you have more granular control over which tasks in your dataset get labeled or reviewed, and in which order.
+
+### Prepare unlabeled data with filters
+
+For example, you can create a filter to prepare tasks to be annotated. For example, if you want annotators to focus on tasks in the first 1000 tasks in a dataset that contain the word "possum" in the field "text" in the task data, do the following: 
+```python
+from label_studio_sdk.data_manager import Filters
+
+Filters.create(Filters.AND, [
+    Filters.item(
+        Column.id,
+        Operator.GREATER_OR_EQUAL,
+        Type.Number,
+        Filters.value(1)
+    ),
+        Filters.item(
+        Column.id,
+        Operator.LESS_OR_EQUAL,
+        Type.Number,
+        Filters.value(1000)
+    ),
+    Filters.item(
+        Column.data(text),
+        Operator.CONTAINS,
+        Type.String,
+        Filters.value("possum")
+    )
+])
+```
+
+### Manage annotations with filters and the Label Studio SDK
+
+For example, to create a filter that displays only tasks with an ID greater than 42 or that were annotated between November 1, 2021, and now, do the following:
+```python
+from label_studio_sdk.data_manager import Filters
```

Comment:
from label_studio_sdk.data_manager import Filters, Column, Type, Operator

Reply:
done thank you!